wsi/drm/drmdisplay.cpp

```diff
@@ -510,6 +510,9 @@ bool DrmDisplay::Commit(
     return false;
   }
 
+  // Disable not-in-used plane once DRM master is reset
```

Comment:
It is better we add a judge before this ResetPlanes call
instead of hiding the judge in ResetPlanes